src/App.tsx

```diff
@@ -1,61 +1,43 @@
+import { useState } from 'react';
+
 import './App.scss';
+import todosFromServer from './api/todos';
+
+import { getUserById } from './services/user';
+import { TodosWithUser } from './types/TodosWithUser';
+
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList';
+
+export const todos: TodosWithUser[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getNewTodoId(todosArray: TodosWithUser[]) {
+  const maxId = Math.max(...todosArray.map(todo => todo.id));
+
+  return maxId + 1;
+}
 
 export const App = () => {
+  const [todo, setTodo] = useState<TodosWithUser[]>(todos);
```

Comment:
The initial state for `todo` is set using `todos`, which is derived from `todosFromServer`. Ensure that `todosFromServer` is correctly formatted and contains all necessary fields.